src/AcceptanceTests/Publishing/When_using_a_signle_bundle.cs

```diff
@@ -32,7 +32,9 @@ public Publisher()
                 EndpointSetup<DefaultServer>(config =>
                 {
                     var transport = config.UseTransport<AzureServiceBusTransport>();
+#pragma warning disable 618
```

Comment:
Would it make sense to assign the topology to var and then only have `NumberOfEntitiesInBundle` in pragma? Since `UseForwardingTopology` is not obsoleted